### PR TITLE
Embedded port map testing and OncRpcEmbeddedPortmap method to check if embedded running

### PR DESCRIPTION
**Pull request by mharj**
Quick testing setup for embedded portmapper

### CODE REVIEW

**Comment by kofemann** [comment]
ok to test.

File: oncrpc4j-core/src/test/java/org/dcache/xdr/OncRpcEmbeddedPortmapTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/*
```
Comment (kofemann) [inline oncrpc4j-core/src/test/java/org/dcache/xdr/OncRpcEmbeddedPortmapTest.java]: can you remove this lines?

Comment (mharj) [inline oncrpc4j-core/src/test/java/org/dcache/xdr/OncRpcEmbeddedPortmapTest.java]: oops, yeah removed :)

Comment (mharj) [inline oncrpc4j-core/src/test/java/org/dcache/xdr/OncRpcEmbeddedPortmapTest.java]: https://github.com/mharj/oncrpc4j/commit/31ab703d63737f06796f521934b81fd9c83f9ca9

**Pull request merged**